torch/ao/quantization/_quantize_pt2e.py

```diff
@@ -0,0 +1,76 @@
+import torch
+from torch.fx import GraphModule
+import torch._dynamo as torchdynamo
+
+from .qconfig_mapping import QConfigMapping
+from .backend_config import BackendConfig
+from .fx import prepare
+from .quantize_fx import _convert_to_reference_decomposed_fx
+from .pt2e.utils import (
+    _get_renamed_nn_module_stack,
+    _fuse_conv_bn_,
+    _rearrange_weight_observer_for_addmm,
+)
+
+import copy
+from typing import Tuple, Any, Dict
+
+# TODO: longer term, don't need to retrace or parse the string
```

Comment:
this is not moved to utils.py yet since I hope to remove it after Sherlock's PR is landed